dist/tools/avarice/debug.sh

```diff
@@ -3,10 +3,10 @@
 # The setsid command is needed so that Ctrl+C in GDB doesn't kill avarice
 : ${SETSID:=setsid}
 
-if gdb-multiarch -v > /dev/null; then
-    GDB=gdb-multiarch
-elif avr-gdb -v > /dev/null; then
+if avr-gdb -v > /dev/null; then
     GDB=avr-gdb
+elif gdb-multiarch -v > /dev/null; then
+    GDB=gdb-multiarch
```

Comment:
Why no longer defaulting to `gdb-multiarch`?

Reply:
I don't know the why, at Debian 10 I can't start gdb with gdb-multiarch only using avr-gdb.

Reply:
I see

Reply:
I can drop the change, it is a proposal and, probably, out of scope.

Reply:
I think it makes sense to get debugging working on as many platforms as possible. If you could split out that commit into a separate PR since this will also affect Debian users with the upstream AVR boards, this would be nice.